doc/source/cluster/running-applications/autoscaling/index.md

```diff
@@ -0,0 +1,54 @@
+(autoscaling)=
+# Autoscaling
+
+The Ray autoscaler is a Ray cluster process that automatically scales a cluster up and down based on resource demand.
+The autoscaler does this by adjusting the number of nodes in the cluster based on the resources required by tasks, actors or placement groups.
+
+:::{note}
+Autoscaling is supported only with Ray versions at least
+as new  as 1.11.0.
```

Comment:
This part accidentally got copied over from the KubeRay docs.
Autoscaling with KubeRay is only supported with Ray versions at least as new as 1.11.0.
Autoscaling on VMs is of course supported since time immemorial...